### PR TITLE
[dualtor] Add active-active ToRs test plan

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Add dualtor active-active mode test plane.

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

### CODE REVIEW

File: docs/testplan/dual_tor/dual_tor_active_active_test_plan.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+| ToR(A) BGP shutdown/startup         | Verify packet flow when admin shutdown/startup BGP sessions on ToR A                             | Admin shutdown/startup the BGP sessions on ToR A                     | ToR A becomes standby during BGP down and change back to active after BGP sessions get re-established, ToR B stays active    | During ToR A is standby(BGP sessions down), T1 switch receives packet only from ToR B(active), after ToR A changes back to active(BGP sessions get up), T1 switch could receive packets from either ToR A or ToR B |
+| ToR(A) reboot                       | Verify packet flow during/after ToR A reboot                                                     | Reboot ToR A                                                         | ToR A becomes standby during reboot and change back to active after reboot, ToR B stays active                               | During ToR A is standby, T1 switch receives packet only from ToR B(active), after ToR A changes back to active, T1 switch could receive packets from either ToR A or ToR B                                         |
```
Comment (wangxin) [inline docs/testplan/dual_tor/dual_tor_active_active_test_plan.md:53]: What if the reboot is 'warm-reboot' or 'fast-reboot'?

File: docs/testplan/dual_tor/dual_tor_active_active_test_plan.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| ToR(A) heartbeat loss               | Verify packet flow after ToR A stops sending heartbeats                                          | Stop the link prober submodule on ToR A                              | ToR A becomes standby and unhealthy link, ToR B stays active                                                                 | T1 switch receives packet from ToR B(active) but not ToR A(standby)                                                                                                                                                |
+| ToR(A) BGP shutdown/startup         | Verify packet flow when admin shutdown/startup BGP sessions on ToR A                             | Admin shutdown/startup the BGP sessions on ToR A                     | ToR A becomes standby during BGP down and change back to active after BGP sessions get re-established, ToR B stays active    | During ToR A is standby(BGP sessions down), T1 switch receives packet only from ToR B(active), after ToR A changes back to active(BGP sessions get up), T1 switch could receive packets from either ToR A or ToR B |
```
[the remark this answers is not in the document]
Comment (lolyu) [reply docs/testplan/dual_tor/dual_tor_active_active_test_plan.md:52]: Yes, to simulate that all default routes are gone

File: docs/testplan/dual_tor/dual_tor_active_active_test_plan.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| ToR A link drop                     | Verify packet flow if the link from ToR A to the server NiC starts dropping packets, but stay up | Configure the server NiC simulator to drop packets destined to ToR A | ToR A becomes standby and unhealthy link, ToR B stays active                                                                 | T1 switch receives packet from ToR B(active) but not ToR A(standby)                                                                                                                                                |
+| ToR(A) heartbeat loss               | Verify packet flow after ToR A stops sending heartbeats                                          | Stop the link prober submodule on ToR A                              | ToR A becomes standby and unhealthy link, ToR B stays active                                                                 | T1 switch receives packet from ToR B(active) but not ToR A(standby)                                                                                                                                                |
```
[the remark this answers is not in the document]
Comment (lolyu) [reply docs/testplan/dual_tor/dual_tor_active_active_test_plan.md:51]: It is elaborated in the `Goal` as ` Verify packet flow if the link from ToR A to the server NiC`

File: docs/testplan/dual_tor/dual_tor_active_active_test_plan.md (hunk 1 above)
Comment (lolyu) [reply docs/testplan/dual_tor/dual_tor_active_active_test_plan.md:53]: `warm-reboot` and `fast-reboot` are not covered here

**Comment by lolyu** [comment]
@wangxin, updated, could you please take another review?

**Review by wangxin — APPROVED** [review]

**Pull request merged**